specs/flipper/flipper.md

```diff
@@ -0,0 +1,105 @@
+# Flipper
+
+## Overview
+
+The flipper component is most often used to page through blocks of content or collections of ui elements.
+
+### Use Cases
+- Tony is looking for a new refrigerator. He goes onto his computer and navigates the browser to a site that sells appliances. He sees a list of refrigerators but not the one he wants so he clicks an arrow button (flipper) to show more. He sees a new GE Cafe that interests him and clicks on it which directs him to a purchase page.
+
+- Bobby is blind and is also looking for a new refrigerator. He goes onto his computer and navigates the browser to the same site that sells appliances. He tabs through a list of refrigerators until he finds the one that was recommended to him by a friend. Bobby is unaware of the flipper.
+
+### Non-goals
+- Other button implementations
+
+### Prior Art/Examples
+Limited prior art is available from major component libraries, however many brands and products leverage flippers on the front page of their sites:
+- Hulu
+- Netflix
+- Amazon
+- Microsoft
+
+---
+
+
+### API
+
+Extends FAST Element
+
+*Component Name*
+- `fast-flipper`
+
+*Attrs*
+- `hiddenFromAT` - The control is hidden from assistive technology. Defaults to true
+- `disabled` - the control is disabled. There are scenarios where a flipper would be visible and disabled
+- `label` - the accessible label for the control, exposed to AT when not hidden
```

Comment:
I think a first class prop here makes sense for label as most of the time flippers do not use text content but visual representations only. Alternatively, we could remove this and just support users passing an aria-label and document that somewhere.

Reply:
I'd +1 for just using an `aria-label`. I don't know that exposing a distinct attribute that then maps the value to the aria-label attribute is necessary. It is a scenario we should cover in documentation though for sure.